README.md

```diff
@@ -43,17 +97,80 @@ None
 Example Playbook
 ----------------
 
+All vars defined, for reference:
+
 ```yaml
 - hosts: reboot-servers
   roles:
     - role: oasis-roles.reboot
-      reboot_seconds_before_reboot: "2"
-      reboot_wait_for_connection_delay: "60"
-      reboot_wait_for_connection_timeout: "300"
+  vars:
+      # Override the underlying connection plugin's connection timeout value,
+      # setting it to 10 seconds
+      reboot_connect_timeout: 10
+      # Show a custom message to users on the system being rebooted
+      reboot_msg: 'A custom message displayed to users on the system being rebooted.'
+      # Give the system 30 seconds to quiesce after rebooting.
+      reboot_post_delay: 30
+      # Wait two minutes before requesting reboot
+      reboot_pre_delay: 120
+      # Find the "shutdown" command in very strange places
+      reboot_search_paths:
+        - '/opt/customshutdown/bin'
+        - '/tmp/'
+      # Test responsiveness with the "uptime" command instead of the default "whoami"
+      reboot_test_command: uptime
+      # Fail if the system is not responsive without 300 seconds of issuing the reboot
+      reboot_timeout: 300
+      # Become the "reboot_user" instead of root to issue the reboot command
+      reboot_become: true
+      reboot_become_user: reboot_user
+```
+
+Rebooting a system when ansible is already a privileged user:
```

Comment:
As a user who is eager to enjoy some copypasta, I wouldn't mind seeing this example at the top of the document since it will be the common, simple use case.

Reply:
Yeah, I think I'll reorder the examples by relevance. Good call. :)